job/nullable_int.go

```diff
@@ -0,0 +1,51 @@
+package job
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// NullableInt represents an int that can be "empty" or nil.
+//
+// A NullableInt is empty by default.
+type NullableInt struct {
+	hasValue bool // false by default
+	i        int
+}
+
+// IsNil returns true if the value is empty.
+func (ni *NullableInt) IsNil() bool {
+	return !ni.hasValue
+}
+
+// Int returns the value, make sure to check ISNil before using Int.
+func (ni *NullableInt) Int() int {
```

Comment:
Rename this to `Value` or `GetValue`. The name NullableInt, already indicates that this structure is an Integer

Reply:
Getters are considered an anti-pattern in Go, see https://golang.org/doc/effective_go#Getters.